index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+require_once './Word.php';
+
+$house = new Word();
+
+$house->name = 'house';
+$house->translation = 'дом';
+$house->transcription = 'haʊs';
+$house->description = 'A house is a single-unit residential building, which may range in complexity from a rudimentary hut to a complex, structure of wood, masonry, concrete or other material, outfitted with plumbing, electrical, and heating, ventilation, and air conditioning systems.';
+$house->examples = [
+                    '1) Montag had rarely seen that many house lights.', 
+                    '2) The house was so small, the furnishings so very simple.'
+                   ];
```

Comment:
1) нет смысла делать так много отступов. читабельность намного лучше не становится, только сложнее станет поддерживать такой код 
2) также хорошей практикой считается добавлять запятую и после последнего элемента массива - это делает более читабельными последующие добавления элементов в него

```suggestion
$house->examples = [
    'Montag had rarely seen that many house lights.',
    'The house was so small, the furnishings so very simple.',
];
```